packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -105,18 +112,27 @@ module.exports = function(env) {
 
 		module: {
 			rules: [
+				{
+					enforce: 'pre',
+					test: /\.tsx?$/,
+					loader: 'awesome-typescript-loader',
+					options: {
+						silent: true,
+						useBabel: true,
```

Comment:
I think this will break BabelEsmPlugin, which looks for babel-loader. 

Would it be possible to run awesome-ts-loader as `{enforce:'pre'}` and then pass `/\.[tj]sx?/` files to babel-loader after?

Reply:
hmm good question